doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -39,10 +39,19 @@ Tactics for simple equalities
 
 .. tacn:: reflexivity
 
-   For a goal with the form :n:`{? forall @open_binders , } t = u`,
-   verifies that `t` and `u` are
-   :term:`definitionally equal <definitional equality>`, and if so,
```

Comment:
Don't remove the reference to definitional equality, which is a formally defined notion in the foundations of the system. This is precisely what explains your first TODO. AFAIR we do describe the generators of definitional equality in the documentation for the typing rules of CIC.

Reply:
I'll fit it in.

Two questions:
1. `Tactics.reflexivity_red` calls `maybe_betadeltaiota_concl` but from examples I see `reflexivity` also does α and ζ reduction.  Is it just that the function name is misleading?
2. [Elsewhere ](http://localhost:63342/coq2/_build/default/doc/refman-html/proofs/writing-proofs/equality.html#term-definitional-equality)the doc says "We say that two terms and 
 are βδιζη-convertible, or simply convertible, or [definitionally equal](http://localhost:63342/coq2/_build/default/doc/refman-html/proofs/writing-proofs/equality.html#term-definitional-equality), in the global environment ..."  Should this sentence also include `α`?

Reply:
1. This is unrelated, this reduction is the one used to find the head equality `M = N`. The algorithm used to convert  `M` and `N` is the one used by the `constructor` tactic, and AFAIK this is some form of unification. It does perform full conversion on ground terms but it may be weaker otherwise.

2. Somehow α is not even really part of the equational theory due to the fact we use De Bruijn indices. You could argue it's thus a non-object, but from the point of view of the user it's as if we had α-conversion everywhere.